.github/workflows/stale.yml

```diff
@@ -13,4 +13,7 @@ jobs:
         stale-issue-label: 'no-issue-activity'
         days-before-stale: 30
         days-before-close: 99999
-        exempt-issue-label: 'enhancement'
+        exempt-issue-label:
+          - 'enhancement'
+          - 'needinfo'
```

Comment:
It is correct for enhancement and bug label; but incorrect for needinfo label.

If a github issue is created and our dev mark it with "needinfo" label; then after stale period, customers doesn't provide required info, this issue should be marked as stale.

If a github issue is created, no one from our team is reviewed => no action taken; therefore even it is longer than stale period, this issue should not be marked as stale.  This case doesn't support in current github stale action.

It is good to have "bug" and "enhancement" for now.  I can see that there is more customized integration with github, e.g. enforce PR commit description.  I think we should think about an extensible solution to address them.